apps/prairielearn/src/pages/partials/navbar.ejs

```diff
@@ -271,24 +271,35 @@
           %>
             <h6 class="dropdown-header">Effective user</h6>
 
-            <div class="dropdown-item-text d-flex flex-row justify-content-left ml-4 align-items-center">
-              <div class="px-0"><input type="email" class="form-control form-control-sm" id="navbar-user-effective-uid-input" placeholder="netid@illinois.edu" style="min-width:15em"></div>
-              <div class="pl-2"><button type="button" class="btn btn-primary btn-sm text-nowrap" id="navbar-user-effective-uid-button" disabled>Change UID</button></div>
-            </div>
+            <form class="form-inline dropdown-item-text d-flex js-effective-uid-form">
+              <label class="sr-only" for="effective-uid">UID</label>
```

Comment:
The `for` attribute requires the value to refer to an ID, not an input name. So either you need to give the input an ID, or you can add the input as a child of the label:

```html
<label>
  <span class="sr-only">UID</span>
  <input ...>
</label>
```

Reply:
Changed to an ID in 5c99d61.